tools/translations/de.po

```diff
@@ -536,31 +536,31 @@ msgstr ""
 
 #: tools/editor/project_export.cpp
 msgid "Export.."
-msgstr ""
+msgstr "Exportieren.."
 
 #: tools/editor/project_export.cpp
 msgid "Project Export"
-msgstr ""
+msgstr "Projekt exportieren"
 
 #: tools/editor/project_export.cpp
 msgid "Export Preset:"
-msgstr ""
+msgstr "Vorlage exportieren"
```

Comment:
For example here, by ignoring the punctuation you mistranslated the string. It should likely be "Exportvorlage:". I'll fix it as I'm merging the commit manually anyway, but keep that in mind for the next time :)